src/stock-receipt/stock-receipt.jsx

```diff
@@ -139,6 +195,47 @@ const StockReceipt = () => {
 			/>
 		);
 	};
+	const handleCloseModal = () => {
+		setShowModal(false);
+	};
+	
+	const handleAddRow = () => {
+		setRows((prevRows) => [
+		...prevRows,
+		{ id: prevRows.length + 1, drugName: "", batchNo: "", expiryDate: "", quantity: 0, totalQuantity: 0 },
+		]);
+	};
+
+	const handleDeleteRow = (id) => {
+	setRows((prevRows) => prevRows.filter((row) => row.id !== id));
+	};
+
+	const handleComboBoxChange = (rowId, selectedValue) => {
+		setRows((prevRows) => {
+		  return prevRows.map((row) => {
+			if (row.id === rowId) {
+			  return { ...row, drugName: selectedValue };
+			}
+			return row;
+		  });
+		});
+	  };
+	
+	const handleInputChange = (id, field, value) => {
+		console.log("value");
+		console.log(value);
```

Comment:
Here aswell we can remove these console.logs

Reply:
Done